src/controllers/IBackendController.ts

```diff
@@ -0,0 +1,20 @@
+import { Express } from "express";
+import { ParamsDictionary, RequestHandler } from "express-serve-static-core";
+
+/*
+ * Standard interface for Backend controllers
+ */
+export interface IBackendController {
+  /**
+   * Method used for seting up routing for Controller
+   */
+
+  // tslint:disable-next-line: no-any
+  readonly setupRouting: <ResBody = any, ReqBody = any>(
+    app: Express,
+    basePath: string,
```

Comment:
You should add a comment here stating that `basePath` should not include a trailing slash

Reply:
Right, should I also validate the variable?